logstash-input-cloudwatch_logs.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_runtime_dependency 'logstash-core-plugin-api', '>= 1.60', '<= 2.99'
-  s.add_runtime_dependency 'logstash-mixin-aws', '>= 4.3.0'
+  s.add_runtime_dependency 'logstash-mixin-aws', '>= 5.1.0'
   s.add_runtime_dependency 'stud', '~> 0.0.22'
 
-  s.add_development_dependency 'logstash-devutils'
+  #s.add_development_dependency 'logstash-devutils'
```

Comment:
Can we delete this?

Reply:
It is dev and can be useful if we were to work on this plugin for big changes but i don't see us changing much that would need it setup

Reply:
I'm just asking because it's commented out. If we do want to keep it that's fine, but then I figure we should uncomment it